CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ IF (NOT RPMDB_LIBRARY)
 ENDIF (NOT RPMDB_LIBRARY)
 FIND_LIBRARY (RPMIO_LIBRARY NAMES rpmio)
 IF (RPMIO_LIBRARY)
-  SET(RPMDB_LIBRARY ${RPMIO_LIBRARY} ${RPMDB_LIBRARY})
+  SET(RPMDB_LIBRARY ${RPMIO_LIBRARY} ${RPMDB_LIBRARY} sqlite3)
```

Comment:
This is not appropriate. Please separately declare SQLite3 as a dependency.

A sample snippet of how to do it:

```cmake
pkg_check_modules(SQLite3 sqlite3 REQUIRED)

...

TARGET_LINK_LIBRARIES(foo ... ${SQLite3_LIBRARIES})
```

In this case, the `pkg_check_modules` would be in the main CMakeLists.txt and the addition of `${SQLite3_LIBRARIES}` in the `TARGET_LINK_LIBRARIES` calls would be in any of the CMakeLists.txt files in subdirectories that use that call.